test/smokes/tslint/expectations.rb

```diff
@@ -217,7 +217,7 @@
     guid: "test-guid",
     timestamp: :_,
     type: "failure",
-    message: "Invalid configuration in `sideci.yml`: unexpected value at config: `$.linter.tslint.config`",
+    message: "The value of the attribute `$.linter.tslint.config` is invalid.",
```

Comment:
[IMO] Users would understand more easily if the config filename would be included...? 🤔 

```suggestion
    message: "The value of the attribute `$.linter.tslint.config` of `sideci.yml` is invalid.",
```

Reply:
I agree. I'll update the message.